### ISSUE TITLE
Custom Alert Notification action

### ISSUE THREAD

**Issue by pbrink231** [Budibase/budibase#7371]
**Describe the feature request**
I would like to either have an action that can create an alert popup.
Currently this already exists but built into the actions with predefined alerts.

I think a simple action with a bindings text box would be amazing and give very simple and easy custom feedback.
could maybe add an icon field to pick one of the icons already in the system.

With this I would also like to have an option to turn off the current alert.

**Screenshots**
![image](https://user-images.githubusercontent.com/8224560/185691658-72c11c7d-455a-41b3-9002-f1ae5a474204.png)

**Comment by JayP718** [comment]
Yes please! I think multiple folks have requested a similar feature, it would be a great enhancement

**Comment by pbrink231** [comment]
I saw one but I think they were thinking about modals.  I have been searching hard for other issues and was surprised I could not find one.

**Comment by JayP718** [comment]
I think they are posted in discussions.

https://github.com/Budibase/budibase/discussions/3396
https://github.com/Budibase/budibase/discussions/5234

And a comment here
https://github.com/Budibase/budibase/discussions/1176

Here
https://github.com/Budibase/budibase/issues/3670

I think there's more but that's all I quickly found.

**Comment by pbrink231** [comment]
Updated answer based on adobe spectrum toast element

**Comment by pbrink231** [comment]
Took a stab at the edits needed.  Did not test these updates anyone is welcome to pickup where I left off.  I dont have a dev system setup yet.
https://github.com/pbrink231/budibase/tree/feature/toast-action

Updating main comment with updated description of what I am looking for

**Comment by aptkingston** [comment]
@pbrink231 I added this over the weekend when I had some free time, and I had to do a double take when I saw your code! I was sure I must have pushed up the code already and you had copied me - but I hadn't pushed it up yet.

Your implementation is pretty much perfect - really impressive stuff.

I've opened a PR there to add this action :+1: The only thing I didn't add was a custom icon, because we don't actually have a nice spectrum icon picker at the moment and I felt it was the least important setting anyway.
